.github/workflows/integration-tests-release.yaml

```diff
@@ -0,0 +1,219 @@
+---
+name: Integration Tests and Release
+
+on: 
+  pull_request:
+    paths:
+      - 'pkg/**'
+      - 'cmd/**'
+      - 'charts/**'
+      - 'tests/integration/**'
+      - '.github/workflows/**'
+  push:
+    branches:
+      - 'main'
+      - '**-run-ci'
+      - '*.*'
+    paths:
+      - 'pkg/**'
+      - 'cmd/**'
+      - 'charts/**'
+      - 'tests/integration/**'
+      - '.github/workflows/**'
+  release:
```

Comment:
Are we using release publishing from Github instead of just pushing a tag?

Reply:
yes, I've explained this in the PR description. Otherwise we tend to forget to publish the GH releases and update the description there.